x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_bulk_delete.test.tsx

```diff
@@ -159,22 +150,27 @@ describe.skip('Rules list bulk delete', () => {
     } as unknown as IToasts;
   });
 
-  beforeEach(() => {
-    wrapper.find('[data-test-subj="checkboxSelectRow-1"]').at(1).simulate('change');
-    wrapper.find('[data-test-subj="selectAllRulesButton"]').at(1).simulate('click');
-    // Unselect something to test filtering
-    wrapper.find('[data-test-subj="checkboxSelectRow-2"]').at(1).simulate('change');
-    wrapper.find('[data-test-subj="showBulkActionButton"]').first().simulate('click');
+  afterEach(() => {
+    jest.clearAllMocks();
+    queryClient.clear();
+    cleanup();
   });
 
-  it('can bulk delete', async () => {
-    wrapper.find('button[data-test-subj="bulkDelete"]').first().simulate('click');
-    expect(wrapper.find('[data-test-subj="rulesDeleteConfirmation"]').exists()).toBeTruthy();
-    wrapper.find('button[data-test-subj="confirmModalConfirmButton"]').simulate('click');
+  beforeEach(async () => {
+    renderWithProviders(<RulesList />);
```

Comment:
I have a feeling this might cause timeouts, but we'll wait and see, if it does I think we can just render once and reset the state using `fireEvent.click` to unselect all the rules, etc.